src/Table.tsx

```diff
@@ -169,11 +169,11 @@ export interface TableProps<Row, Key> extends Omit<StandardProps, 'onScroll'> {
    * whether to update the scroll after the table size is updated.
    */
   shouldUpdateScroll?:
-    | boolean
-    | ((event: TableSizeChangeEventName) => {
-        x?: number;
-        y?: number;
-      });
+  | boolean
```

Comment:
You can run the following command locally to check if your code complies with the specification:

```bash
npm run lint
npm run test
```